CoreFoundation/PlugIn.subproj/CFBundle_Executable.c

```diff
@@ -303,38 +275,96 @@ static CFURLRef _CFBundleCopyExecutableURLInDirectory2(CFBundleRef bundle, CFURL
     return executableURL;
 }
 
+static CFStringRef _CFStringRemovePrefixSuffix(CFStringRef str, CFStringRef prefix, CFStringRef suffix) {
```

Comment:
This needs to have a `Create` in the name.

Reply:
Maybe `_CFBundleCopyBundleNameByRemovingExcutablePlatformPrefixSuffix`?